lib/browser/api/web-contents.ts

```diff
@@ -543,6 +543,11 @@ WebContents.prototype._init = function () {
 
   this.on('crashed', (event, ...args) => {
     app.emit('renderer-process-crashed', event, this, ...args);
+
+    // Log out a hint to help users better debug renderer crashes.
+    if (!process.env.ELECTRON_ENABLE_LOGGING) {
```

Comment:
* (nit) Electron's logging doesn't care if the value of `ELECTRON_ENABLE_LOGGING` is truthy or not; only that it exists. So it's more accurate to see if process.env hasOwnProperty named `ELECTRON_ENABLE_LOGGING`

* (not nit) We also need to check for `--enable-logging` which is currently supported according to docs, and is _definitely_ supported in the copy-logging-from-headless branch.

Possibly overkill, but more accurate:

```js
function isLogging () { 
  return Object.prototype.hasOwnProperty.call(process.env, 'ELECTRON_ENABLE_LOGGING') ||
    process.argv.some(arg => arg.toLowerCase().startsWith('--enable-logging'));
}
```

(The startsWith() check is because the copy-headless-logging PR allows for --enable-logging=foo)